### PR TITLE
Camera exposure example

### PR DESCRIPTION
**Pull request by sjkummer**
**Description**
Added an example for exposure control based on getusermedia/ptz. 
Tested on Windows with Logitech Brio and C525 using official Logitech drivers.

**Purpose**
- The support for exposure controls differs between cameras, platforms and drivers. It's nice to have an easy way to test it.
- Keep @fippo busy 😉

### CODE REVIEW

File: src/content/getusermedia/exposure/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
```
Comment (fippo) [inline src/content/getusermedia/exposure/index.html]: nit: update to 2022

File: src/content/getusermedia/exposure/js/main.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+/*
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
```
Comment (fippo) [inline src/content/getusermedia/exposure/js/main.js]: same

File: src/content/getusermedia/exposure/js/main.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+'use strict';
+
+// Put variables in global scope to make them available to the browser console.
+const constraints = window.constraints = {
+  video: {
+    pan: true, tilt: true, zoom: true
```
Comment (fippo) [inline src/content/getusermedia/exposure/js/main.js]: this was copied from the PTZ sample? Remove if not needed please

**Comment by fippo** [comment]
The more samples the merrier!

eslint should be able to fix most errors on its own

File: src/content/getusermedia/exposure/index.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    </div>
+	
+	<div>
+        <div class="label">Brightness:</div>
+        <input name="brightness" type="range" disabled>
+    </div>
+	
+	<div>
+        <div class="label">White Balance Mode:</div>
+        <select name="whiteBalanceMode" id="whiteBalanceMode" disabled>
```
Comment (sjkummer) [inline src/content/getusermedia/exposure/index.html:71]: I realized that `whiteBalanceMode` is out of scope. Should I remove it @fippo ?

**Comment by sjkummer** [comment]
> eslint should be able to fix most errors on its own

eslint should be fine now

**Comment by sjkummer** [comment]
Seems like the chromedriver version is outdated in package.json:
```"chromedriver": "^98.0.1",```

CI:
```Current browser version is 100.0.4896.75 with binary path browsers/bin/chrome-stable```


Should I change that, or would you prefer to fix that in a separate PR? @fippo

**Comment by fippo** [comment]
that, along with some flakyness is fixed in #1534

File: src/content/getusermedia/exposure/index.html (hunk 4 above)
Comment (fippo) [reply src/content/getusermedia/exposure/index.html:71]: the more knobs the merrier

**Review by fippo — APPROVED** [review]
LGTM (now who gets to fix the reported bug.. :-))

**Comment by pgurusami** [comment]
@sjkummer 
Tried running in macOs and it reports that the controls are not available.
But with the same webcam, I am able to adjust the settings in Windows system.
your inputs will be helpful

**Comment by fippo** [comment]
Maybe @beaufortfrancois knows the status of camera exposure on MacOS...

**Comment by beaufortfrancois** [comment]
UVC Controls on Chrome for macOS are currently disabled sadly: https://source.chromium.org/chromium/chromium/src/+/main:media/capture/video/mac/uvc_control_mac.mm;l=29;drc=2b892bc0f2a92c512d433f73a68bd1c41c82131b
I'm not aware of plans to re-enable them by default. @handellm may know more

**Pull request merged**